src/20240707_AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity/AaveV3Arbitrum_IncreaseCCIPFacilitatorCapacity_20240707.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3ArbitrumAssets} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+interface IGHOToken {
```

Comment:
Gho lib is part of the repo so I think it's better to directly import the interface

Reply:
done